src/actions/NamespaceAction.ts

```diff
@@ -1,14 +1,13 @@
 import * as Api from '../services/Api';
-import { createAction } from 'typesafe-actions';
+import { ActionType, createAction, createStandardAction } from 'typesafe-actions';
 import Namespace from '../types/Namespace';
 import { KialiAppState } from '../store/Store';
 
-export enum NamespaceActionKeys {
+enum NamespaceActionKeys {
   NAMESPACE_REQUEST_STARTED = 'NAMESPACE_REQUEST_STARTED',
   NAMESPACE_SUCCESS = 'NAMESPACE_SUCCESS',
   NAMESPACE_FAILED = 'NAMESPACE_FAILED',
-  SET_ACTIVE_NAMESPACE = 'SET_ACTIVE_NAMESPACE',
-  SET_PREVIOUS_GRAPH_STATE = 'SET_PREVIOUS_GRAPH_STATE'
```

Comment:
I thought I had already removed this in my activeNamespace work last week. Thanks for killing it.

Reply:
At some point it will need to added back in because we need to remove the `session`. But later PR.